cmd/vela-server/main.go

```diff
@@ -175,6 +175,23 @@ func main() {
 			Usage:   "source commit status context",
 			Value:   "continuous-integration/vela",
 		},
+
+		&cli.StringFlag{
```

Comment:
Would these flags ever need to be arrays? I'm trying to understand if an administrator would ever want to maintain more than one modification system.

Reply:
I'd opt to only allow a single modification system until we determine the use case for multiple. I see a potential use case for having different types of modification systems that are called at different points of compilation, but that is likely outside the scope of the initial implement ion IMO.